CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [4.0.1](https://github.com/folio-org/ui-checkout/tree/v4.0.1) (2020-06-19)
 [Full Changelog](https://github.com/folio-org/ui-checkout/compare/v4.0.0...v4.0.1)
 
+* Checkout barcode CQL injection.  Fixes UICHKOUT-633.
```

Comment:
`4.0.1` has already been published. Instead, start a new release at the top like this: 

```
## 4.1.0 (IN PROGRESS)

* Checkout barcode CQL injection. Fixes UICHKOUT-633.
* next change
* and the next one after that...

```